src/sidebar/util/annotation-metadata.js

```diff
@@ -3,6 +3,7 @@
  */
 
 /** @typedef {import('../../types/api').Annotation} Annotation */
+/** @typedef {import('../../types/annotator').AnnotationData} AnnotationData */
```

Comment:
The changes in this file are not needed any more because of the changes you made to the place where `isHighlight` is called.

Reply:
yes, it only needs Annotation now, good catch